Exercises.py

```diff
@@ -23,23 +22,23 @@
 # 4. Checking Element Presence:
 # Create a list and ask the user to input a number, then check if this number is present in the list.
 
-l = [1, 2, 3]
-print(l[1])
-if l[1] == 2:
-    print('number is present')
+list_numbers = [1, 2, 3]
+user_number = int(input("Please enter the number: ")) # input() function
+if user_number in list_numbers:
```

Comment:
only check if number is in a list
result should be of type boolean

Reply:
fixed